fairlogger.sh

```
@@ -6,7 +6,8 @@ requires:
  - fmt
 build_requires:
  - CMake
- - "GCC-Toolchain:(?!osx)"
+ - GCC-Toolchain
+ - alibuild-recipe-tools
 incremental_recipe: |
   cmake --build . --target install ${JOBS:+-- -j$JOBS}
   mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)